deploy-finder-process.md

```diff
@@ -11,31 +11,36 @@ This will enqueue the documents via Sidekiq.
 
 * Once inside backend integration go to: `cd /var/apps/specialist-publisher`
 
-* Check size of the Sidekiq queue using: `Queue.all.first.size`
+* Within `govuk_app_console specialist-publisher` check size of the Sidekiq queue using: `Sidekiq::Queue.all.first.size`
```

Comment:
Good spot. In our case, that's ok because we're only interested in the `bulk_republishing` queue, which is the first queue:

https://github.com/alphagov/specialist-publisher/blob/master/config/sidekiq.yml#L3